File: ingestion_service/tests/fixtures/python_repo_valid/pkg/animal.py
from .movable import Movable
from .util.helpers import calc


class Animal(Movable):
    def speak(self):
        return "..."

    def describe(self):
        return self.speak()

    def move_to(self, x, y):
        self.speak()

    def scaled(self, x):
        return calc(x)

File: ingestion_service/tests/fixtures/java_repo/src/main/java/com/acme/Animal.java
package com.acme;

import com.acme.util.Util;
import com.acme.other.*;

public class Animal implements Movable {
    private String name;

    public Animal(String name) {
        this.name = name;
    }

    public String speak() {
        return name;
    }

    public String describe() {
        return speak();
    }

    @Override
    public void moveTo(int x, int y) {
        speak();
    }

    public int scaled(int x) {
        return Util.calc(x);
    }

    public void overload(int x) {
    }

    public void overload(String x) {
    }

    public class Tag {
        public void tagMethod() {
        }
    }
}
